api/v1/models/wireguard_interface.go

```diff
@@ -38,6 +38,9 @@ type WireguardInterface struct {
 
 	// Public key of this interface
 	PublicKey string `json:"public-key,omitempty"`
+
+	// Node Encryption status
+	NodeEncryption string `json:"node-encryption,omitempty"`
```

Comment:
It seems like this was added without changing the swapper API. Please apply this modification to 

https://github.com/cilium/cilium/blob/02ef3417903e5acbd0d8cddd7ef3248bc366569e/api/v1/openapi.yaml#L3380-L3385 

And then re-generate this file via `make generate-api`.